components/App.js

```diff
@@ -24,91 +24,21 @@ import {
   ReloadInstructions,
 } from 'react-native/Libraries/NewAppScreen';
 
+import ItemsList from './ItemsList'
+import { styles } from './styles/stylesItemsList'
+
+
 const App: () => React$Node = () => {
+  const object = {
+    Name: "Alex", Age: 30, City: "TLV", Hobbies: ["reading", "sport", "movies"]
```

Comment:
a bit weird that this is here this object has nothing to do with app.js

Reply:
So where would you suggest to put it? I didn't really understood where the object presented is coming from